comp22/score-sheet-2022.svg

```diff
@@ -0,0 +1,517 @@
+         id="rect3157-7-7" />
+      <rect
+         style="fill:#ffffff;stroke:#000000;stroke-width:2.37467;stroke-linecap:square;stroke-linejoin:miter;stroke-miterlimit:4;stroke-dasharray:none;stroke-dashoffset:0;stroke-opacity:1"
+         id="rect3900"
+         width="144.67368"
+         height="144.67368"
+         x="674.51422"
+         y="50.92448"
+         transform="matrix(0.72162819,0.69228083,-0.72162819,0.69228083,0,0)" />
+    </g>
+    <text
+       xml:space="preserve"
+       style="font-style:normal;font-variant:normal;font-weight:bold;font-stretch:normal;line-height:0%;font-family:'Open Sans';-inkscape-font-specification:'Open Sans';letter-spacing:0px;word-spacing:0px;fill:#000000;fill-opacity:1;stroke:none"
+       x="440"
+       y="77.569199"
+       id="text3408"><tspan
+         id="tspan3410"
+         x="440"
+         y="77.569199"
+         style="font-size:28px;line-height:1.25">2022 Score Sheet</tspan></text>
+    <text
+       id="text3523"
+       y="406.36215"
+       x="128.51953"
+       style="font-style:normal;font-variant:normal;font-weight:800;font-stretch:normal;line-height:0%;font-family:'Rockwell Extra Bold';-inkscape-font-specification:'Rockwell Extra Bold, Ultra-Bold';letter-spacing:0px;word-spacing:0px;fill:#000000;fill-opacity:1;stroke:none"
+       xml:space="preserve"><tspan
+         y="406.36215"
+         x="128.51953"
+         id="tspan3525"
+         style="font-size:28px;line-height:1.25"> </tspan></text>
+    <use
+       x="0"
+       y="0"
+       xlink:href="#text3523"
+       id="use3568"
+       transform="translate(403.48047,-13.000002)"
+       width="744.09448"
+       height="1052.3622" />
+    <text
+       xml:space="preserve"
+       style="font-style:normal;font-variant:normal;font-weight:bold;font-stretch:normal;font-size:9.37386px;line-height:0%;font-family:'Open Sans';-inkscape-font-specification:'Open Sans';text-align:center;letter-spacing:0px;word-spacing:0px;text-anchor:middle;fill:#000000;fill-opacity:1;stroke:none;stroke-width:0.781155"
+       x="161.94299"
+       y="133.73578"
+       id="text3412-5-0"><tspan
+         x="161.94299"
+         y="133.73578"
+         id="tspan3497-4-0"
+         style="font-size:21.8723px;line-height:1.25;stroke-width:0.781155">Match Number: _______</tspan></text>
+    <text
+       xml:space="preserve"
+       style="font-style:normal;font-variant:normal;font-weight:bold;font-stretch:normal;font-size:9.37386px;line-height:0%;font-family:'Open Sans';-inkscape-font-specification:'Open Sans';text-align:start;letter-spacing:0px;word-spacing:0px;text-anchor:start;fill:#000000;fill-opacity:1;stroke:none;stroke-width:0.781155"
+       x="344.0451"
+       y="133.34596"
+       id="text3412-5-0-0-7"><tspan
+         x="344.0451"
+         y="133.34596"
+         id="tspan3497-4-0-6-3"
+         style="font-size:21.8723px;line-height:1.25;stroke-width:0.781155">Scorer: _______________________________</tspan></text>
```

Comment:
Now there's a bit more space can we reinstate the "name" part here?
In the past we've had confusion over what was meant to go in this field, which was why it included that.

Reply:
2ca94a9b3290672c33831d63fcf93d5474e8ca3b